react/features/base/tracks/actions.any.ts

```diff
@@ -539,10 +535,6 @@ export function trackRemoved(track: any): {
     };
     type: 'TRACK_REMOVED';
 } {
-    track.removeAllListeners(JitsiTrackEvents.TRACK_MUTE_CHANGED);
```

Comment:
Why are we no longer doing this?

Reply:
For the legacy case, this has no effect since the track is already disposed and all listeners removed before the `JitsiConferenceEvents.TRACK_REMOVED` event is fired. 
For the ssrc-rewriting case, the track is not disposed so I tried not removing the listeners to see if the delay goes away but executing play() on the large-video fixed it. I brought this back now.